masterlist.yaml

```diff
@@ -4025,7 +4216,7 @@ plugins:
         name: 'Legacy of the Dragonborn Patches SE on Nexus Mods'
     tag:
       - Actors.AIPackages
-      - -Factions
```

Comment:
The extra hyphen is to signify a bad tag, which should be removed.

Reply:
How do you verify a bad tag?  I noticed the xEdit script still suggested a Factions tag.